disttask/framework/scheduler/manager.go

```diff
@@ -373,11 +374,14 @@ func (m *Manager) removeHandlingTask(id int64) {
 	delete(m.mu.handlingTasks, id)
 }
 
-// onError handles the error.
-func (m *Manager) onError(err error) {
-	if err == nil {
-		return
-	}
-
+func (m *Manager) logErr(err error) {
 	logutil.Logger(m.logCtx).Error("task manager error", zap.Error(err))
 }
+
+func (m *Manager) logErrAndPersist(err error, taskID int64) {
+	m.logErr(err)
```

Comment:
Will here get return value of m.logErr(err), then store return packed err into subtask?

Reply:
It is designed NOT to store errors in subtasks. That's why I change into two methods.